CMakeLists.txt

```diff
@@ -502,7 +502,51 @@ add_library(torrent-rasterbar
 	${libtorrent_aux_include_files}
 )
 
-select_cxx_standard(torrent-rasterbar 11)
+# Full C++ 11 support is required
+target_compile_features(torrent-rasterbar
+	PUBLIC
+		cxx_std_11
+		cxx_rvalue_references
+		cxx_reference_qualified_functions
+		cxx_nonstatic_member_init
+		cxx_variadic_templates
+		cxx_generalized_initializers
+		cxx_static_assert
+		cxx_auto_type
+		cxx_trailing_return_types
+		cxx_lambdas
+		cxx_decltype
+		cxx_decltype_incomplete_return_types
+		cxx_right_angle_brackets
+		cxx_default_function_template_args
+		cxx_alias_templates
+		cxx_extern_templates
+		cxx_nullptr
+		cxx_strong_enums
+		cxx_enum_forward_declarations
+		cxx_attributes
+		cxx_constexpr
+		cxx_alignas
+		cxx_alignof
+		cxx_delegating_constructors
+		cxx_inheriting_constructors
+		cxx_explicit_conversions
+		cxx_raw_string_literals
+		cxx_user_literals
+		cxx_defaulted_functions
+		cxx_deleted_functions
+		cxx_extended_friend_declarations
+		cxx_sizeof_member
+		cxx_inline_namespaces
+		cxx_unrestricted_unions
+		cxx_local_type_template_args
+		cxx_range_for
+		cxx_final
+		cxx_override
+		cxx_noexcept
+		cxx_defaulted_move_initializers
+		cxx_thread_local
```

Comment:
there's actually a work-around in libtorrent when `thread_local` is not available, so this shouldn't be here

Reply:
fixed